indico/web/templates/forms/dropzone_widget.html

```diff
@@ -15,7 +15,7 @@
             </div>
         </div>
     </div>
-    <input class="change-trigger" type="hidden" value="0" name="__file_change_trigger">
+    <input class="change-trigger" type="hidden" value="{{ 1 if field.data else 0}}" name="__file_change_trigger">
```

Comment:
there's still JS that sets this to a random number. not sure if that's still needed (or if the code that sets it to 1 or 0 should be changed instead)